addon/-private/system/model/internal-model.js

```diff
@@ -707,11 +707,11 @@ export default class InternalModel {
     Preloaded data can be attributes and relationships passed in either as IDs or as actual
     models.
 
-    @method _preloadData
+    @method preloadData
     @private
     @param {Object} preload
   */
-  _preloadData(preload) {
```

Comment:
@stefanpenner I removed this one because InternalModel is "entirely private api" and this is used by other private modules, I believe that was our stated api naming goal?